LQResultProducer.py

```diff
@@ -4564,9 +4564,10 @@ def SysTable(optimlog, selection_uujj,selection_uvjj,NormalDirectory, weight,sys
 		exec (_tree+" = "+_treeTmp+".Get(\""+TreeName+"\")")
 	selection_uujj = selection_uujj+'*'+weight
 	selection_uvjj = selection_uvjj+'*'+weight
+	selection_uujj_diboson = NormalWeightDiBoson+'*'+preselectionmumu_3lep
```

Comment:
look inside the ModSelection function. A selection has to have 'weight' in it to get variations. You have to pass the diboson selection and weight (same weight as other samples obviously) through the function. Please avoid any hardcoding of selections or weights inside functions

Reply:
I've added new arguments to the FullAnalysis, SysTable, and ShapeSystematics functions that pass the selection and weights specific to the diboson SF control region. Now instead of hardcoding in each function, we pass NormalWeightDiBoson and preselectionmumu_3lep to FullAnalysis, which get propagated to the Rvv calculations. Both NormalWeightMuMu and NormalWeightDiBoson strings that are propagated to ModSelection via FullAnalysis have 'weight' in them (from 'weight_topPt'). This should address the issue!